apps/src/templates/teacherNavigation/TeacherNavigationBar.tsx

```diff
@@ -41,6 +42,102 @@ const TeacherNavigationBar: React.FunctionComponent = () => {
     }
   }, [sections, selectedSectionId]);
 
+  const courseContent = (
```

Comment:
I honestly feel like these three sections are a bit lengthy, but I didn't see any real benefit from putting them into their own components.  I think having all these here will also make it easier to modify what is selected at any given time.